src/command/feed/feed-command.ts

```diff
@@ -38,6 +38,9 @@ export class FeedCommand extends RootCommand {
   @Option({ key: 'password', alias: 'P', description: 'Password for the wallet' })
   public password!: string
 
+  @Option({ key: 'index', description: 'Index for the feed to write to, or read from', required: false })
```

Comment:
Maybe it could simply say "Feed index to write to or read from"?

Reply:
Good idea. I'll make this change.